python-frontend/src/main/java/org/sonar/plugins/python/api/ProjectPythonVersion.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2011-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.python.api;
+
+public class ProjectPythonVersion {
+
+  private ProjectPythonVersion() {
+  }
+
+  private static PythonVersion currentVersion = PythonVersion.allVersions();
+
+  public static PythonVersion currentVersion() {
+    return currentVersion;
+  }
+
+  public static void setCurrentVersion(PythonVersion currentVersion) {
+    ProjectPythonVersion.currentVersion = currentVersion;
```

Comment:
Related to my other comment, it's kind of weird to me that what actually holds the project python version is the attribute `supportedVersions` of a field of type `PythonVersion` nested in `ProjectPythonVersion`.

I feel it could be a bit simpler, e.g by having the `PythonVersion` class as a fully util class whose `fromString` method would return the `List<Version>` which would be what's actually stored in `ProjectPythonVersion`, wdyt?

Reply:
Agreed, we don't need to abstract Python version in a class for the time being!
I think we can indeed have `PythonVersionUtils.fromString()` returning a `Set<Version>` and `ProjectPythonVersion` storing just this set